src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/ClientCredentialTypeTests.cs

```diff
@@ -34,7 +34,7 @@ public static void BasicAuthentication_RoundTrips_Echo()
 
             ChannelFactory<IWcfCustomUserNameService> factory = new ChannelFactory<IWcfCustomUserNameService>(basicHttpBinding, new EndpointAddress(Endpoints.Https_BasicAuth_Address));
             factory.Credentials.UserName.UserName = "test1";
-            factory.Credentials.UserName.Password = "test1pwd";
+            factory.Credentials.UserName.Password = "Mytestpwd1";
```

Comment:
Non-blocking questions -- why are we changing this and is there a matching string literal somewhere else we have to maintain separately?  If so, we should add some comments or share code (and in a separate PR).